### PR TITLE
Add copy room link and name change options

### LINKED ISSUE #71: Add a way to change your name
On the `/options` page.

### LINKED ISSUE #72: Add a link to the room on the options page
We need a quick way to show the link for people to type or copy and send to others.

### PR DESCRIPTION
**Pull request by jacebrowning**
Closes #71
Closes #72

### CODE REVIEW

**Comment by jacebrowning** [comment]
To test:

- [ ] View the room link on the options page
- [ ] Copy the room link
- [ ] Using the options button try to change your name:
  - [ ] Valid name
  - [ ] Name with extra spaces
  - [ ] Blank name

File: kcards/static/options.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var clipboard = new Clipboard(".btn");
+
+clipboard.on('success', function(e) {
+    $('#flashes').append('<p class="info">Link copied to clipboard.</p>');
```
Comment (jacebrowning) [inline kcards/static/options.js]: For reference: http://stackoverflow.com/questions/24288380/flask-flash-and-url-for-with-ajax

**Comment by DanLindeman** [comment]
All of the functionality is in place, but I have a question on style.

In the options page the Sharing field is small, and the url is cut off. Is there a way we can handle this ti display the whole text, reminiscent of using `backticks` for monospaced markup?

**Comment by DanLindeman** [comment]
We can always just open a front end issue and circle back later.

**Pull request merged**